tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -226,9 +226,19 @@ describe('Test resource behavior after deleting GitRepo using keepResources opti
       // Delete GitRepo to check application removed or not.
       cy.deleteAllFleetRepos();
 
-      // Check applicatio still exists after delting existing GitRepo
+      // Check application still exists after deleting existing GitRepo
       cy.checkApplicationStatus(appName);
       cy.deleteApplicationDeployment();
     })
   )
 });
+
+describe('Test local cluster behavior with New workspace', { tags: '@p1'}, () => {
+  qase(107,
+    it("Fleet-107: Test LOCAL CLUSTER cannot be moved to another workspace as 'NO CHANGE WORKSPACE' option available..", { tags: '@fleet-107' }, () => {
```

Comment:
```suggestion
    it("Fleet-107: Test LOCAL CLUSTER cannot be moved to another workspace as no 'Change workspace' option available.", { tags: '@fleet-107' }, () => {
```

Reply:
Thanks for suggestion, I will update it.